docs/src/content/docs/it/guides/sidebar.mdx

```diff
@@ -237,20 +239,26 @@ La configurazione del frontmatter `sidebar` viene utilizzata solo per i collegam
 
 I collegamenti possono anche includere una proprietà `badge` per visualizzare un badge accanto all'etichetta del collegamento.
 
-```js
+```js {10,17}
 starlight({
 	sidebar: [
 		{
-			label: 'Guide',
+			label: 'Guides',
 			items: [
-				// Un collegamento con un badge "Nuovo".
+				// Un link con il badge "Nuovo".
 				{
-					label: 'Componenti',
+					label: 'Components',
 					link: '/guides/components/',
-					badge: 'Nuovo',
+					badge: 'New',
```

Comment:
We can safely translate labels, do you think in this case it did not make sense?